### ISSUE TITLE
Rename uids in beta_per_dt

### ISSUE THREAD

**Issue by daniel-klein** [starsimhub/starsim#540]
Beta per dt is as follows:
```python
def beta_per_dt(self, disease_beta=None, dt=None, uids=None):
    if uids is None: uids = Ellipsis
    return self.edges.beta[uids] * disease_beta * dt
```

As you can see, "uids" are used to index beta in network edges. These are not uids, but rather rows in the edge list. Consider renaming away from "uids" unless I am misunderstanding.

**Comment by devclinton** [comment]
I would propose
```
def beta_per_dt(self, disease_beta=None, dt=None, rows=None):
```
or
```
def beta_per_dt(self, disease_beta=None, dt=None, indices=None):
```

**Comment by devclinton** [comment]
Alternatively, we could duplicate logic we have for people  for edges to allow referencing previous edges in another timestep

**Comment by cliffckerr** [comment]
This method will mostly likely be removed in the next Starsim release as time-aware parameters and module-specific `dt`s will require a different implementation.

**Comment by cliffckerr** [comment]
Closed by #654
